File: net/split/Split_HTML.cs
using Aspose.Pdf;
using Aspose.Pdf.Facades;

namespace aspose_snippets.net
{
    public static partial class Split
    {
        public static void HTML()
        {
            var pathSource = @"..\..\TestData\test.html";
            using (var doc = new Document(pathSource, new HtmlLoadOptions()))
            {
                //save input html to pdf to file
                doc.Save("test.pdf", SaveFormat.Pdf);
            }
            
            var pdfEditor = new PdfFileEditor();
            pdfEditor.SplitFromFirst("test.pdf", 1, "test.pdf");
            using (var doc = new Document("test.pdf"))
            {
                doc.Save("first_page.html", SaveFormat.Html);
            }

            File.Delete("test.pdf");
        }
    }
}

File: java/split/src/aspose_snippets/java/Split_HTML.java
package aspose_snippets.java;

public class Split_HTML {
    public void Execute() {
        String pathSource = "../../TestData/test.html";
        com.aspose.pdf.Document doc = new com.aspose.pdf.Document(pathSource, new com.aspose.pdf.HtmlLoadOptions());
        //save input html to pdf to file
        doc.save("test.pdf", com.aspose.pdf.SaveFormat.Pdf);
    
        com.aspose.pdf.facades.PdfFileEditor pdfEditor = new com.aspose.pdf.facades.PdfFileEditor();
        //slit first page
        pdfEditor.splitFromFirst("test.pdf", 1, "test.pdf");
        com.aspose.pdf.Document outputDoc = new com.aspose.pdf.Document();
        outputDoc.save("first_page.html", com.aspose.pdf.SaveFormat.Html);
    }
}
